content/src/Environment.ts

```diff
@@ -9,6 +9,7 @@ import { AppComponents } from './types'
 export const CURRENT_CONTENT_VERSION: EntityVersion = EntityVersion.V3
 const DEFAULT_STORAGE_ROOT_FOLDER = 'storage'
 const DEFAULT_SERVER_PORT = 6969
+const DEFAULT_FOLDER_MIGRATION_MAX_CONCURRENCY = 1000
```

Comment:
Is this the max amount of files per folder?

Reply:
No, that's the default concurrency of migrations